src/grz_cli/utils/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""Utility functions and classes for the GRZ CLI."""
+
+from .checksums import calculate_sha256
+from .config import read_config
+from .crypt import Crypt4GH
+from .io import TqdmIOWrapper, read_multiple_json
+from .paths import is_relative_subdirectory
+
+__all__ = [
+    "Crypt4GH",
+    "TqdmIOWrapper",
+    "calculate_sha256",
+    "is_relative_subdirectory",
+    "read_config",
+    "read_multiple_json",
+]
```

Comment:
This seems redundant since this is everything imported.

Reply:
It seems redundant, but otherwise [F401](https://docs.astral.sh/ruff/rules/unused-import/) is triggered.

Reply:
We could also skip the re-exports and just use the full path instead

Reply:
Either that or consider adding ignore statements for those lines if you want to keep them?

Reply:
I removed the re-exports altogether